From: Jakob Haufe <sur5r@sur5r.net>
Date: Thu, 29 Mar 2012 22:20:26 +0200
Subject: Ignore unused screenshot
Bug: https://sourceforge.net/tracker/?func=detail&aid=3515881&group_id=46122&atid=445116

Ignore figures/main-window-empty.png as it's no longer used by the docs
and generates a self-referencing symlink.

This should go upstream
---
 help/Makefile.am |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/help/Makefile.am b/help/Makefile.am
index 4177122..1185a90 100644
--- a/help/Makefile.am
+++ b/help/Makefile.am
@@ -11,7 +11,6 @@ DOC_FIGURES = \
 	figures/glabels-template-layout.png		\
 	figures/glabels-template-rect-label.png		\
 	figures/introduction-glabels-main-window.png	\
-	figures/main-window-empty.png			\
 	figures/merge-ex1-1.png				\
 	figures/merge-ex1-2.png				\
 	figures/merge-ex1-3.png				\
